other/upgrade_2-1_mysql.sql

```diff
@@ -1022,22 +1022,22 @@ if (!empty($select_columns))
 		while ($row = $smcFunc['db_fetch_assoc']($request))
 		{
 			if (!empty($row['aim']))
-				$inserts[] = array($row['id_member'], -1, 'cust_aolins', $row['aim']);
+				$inserts[] = array($row['id_member'], 0, 'cust_aolins', $row['aim']);
```

Comment:
I think you're right...   Let me fix this...
I just added a bunch of these attributes manually online, & they all get added with a theme_id of 1.
I think the data migration should mimic the manual data entry, so I'll set it to 1.